client/src/Login.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
+
+.photo {
+    width: 270px
+}
+
+
+.Logo {
+    display: flex;
+    justify-content: center;
+    padding-top: 3cm;
+}
+
+.inputTitle {
+    text-align: left;
+    font-family: 'Montserrat', sans-serif;
+    color: #81A1B7;
+    font-size: 14pt;
+    font-weight: 700;
+    margin: 5px;
+}
+
+.orText {
+    display: flex;
+    justify-content: center;
+    color: rgb(0, 132, 156);
+    font-weight: 700;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.forgetPass {
+    display: flex;
+    justify-content: center;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
+}
+
+.forgetPass a:link, a:visited {
+    color: rgb(0, 132, 156);
+    font-weight: 400;
+    padding-top: 20px;
```

Comment:
Notice how the "forgot password" text is still clickable 20px above the text. This is because we're adding `padding-top: 20px` instead of `margin-top: 20px`, which is extending the clickable region 20px above the text instead of adding 20px of non-clickable white space. We don't want the white space to be clickable too, so let's switch it over to `margin-top: 20px`.